async-nats/src/service/mod.rs

```diff
@@ -60,6 +59,23 @@ pub(crate) struct Endpoints {
     pub(crate) endpoints: HashMap<String, endpoint::Inner>,
 }
 
+/// Response for `PING` requests.
+#[derive(Serialize, Deserialize)]
```

Comment:
If it's ping response, it's not Ping?

Either `Pong` or `PingResponse`?

Reply:
done, I'll update compat tests PR as well